content/docs/reference/routes/public-access.mdx

```diff
@@ -16,9 +16,24 @@ import TabItem from '@theme/TabItem';
 
 ## Summary
 
-**Use with caution:** **Public Access** allows all requests for a given route, bypassing authentication and authorization. Suitable for publicly exposed web services.
+:::caution
 
-If this setting is enabled, no whitelists (for example, Allowed Users) should be provided in this route.
+Because the **Public Access** setting bypasses authentication and authorization checks, you should only enable it for publicly exposed web services.
```

Comment:
nit:

```suggestion
Because the **Public Access** setting bypasses authentication and authorization checks, you should enable it only for publicly exposed web services.
```

(This might be my <span>#</span>1 writing style nit ever since I became aware of it. See e.g. https://getitwriteonline.com/placement-of-only/ for more on the rationale.)